canonical-websites/www.ubuntu.com
# 2664: Build /openstack section from /cloud section

## Done

- Updated the /openstack page based on this [copydoc](https://docs.google.com/document/d/1eqHK0hc9v7Mn-XV36QRyNkJenTXgBQR_-RULKm5rZ5o/edit#)
- Moved /cloud/contact-us to /openstack/contact-us and created a redirect
- Moved /cloud/partners to /openstack/partners and created a redirect
- Moved /cloud/training to /openstack/training and created a redirect
- Moved /cloud/storage to /openstack/storage and created a redirect
- Moved /cloud/managed-cloud to /openstack/managed-cloud and created a redirect
- Moved /cloud/foundation-cloud to /openstack/foundation-cloud and created a redirect
- Moved /cloud/contact-us to /openstack/contact-us and created a redirect
- Moved /cloud/thank-you to /openstack/thank-you and created a redirect
- Moved /cloud/juju to /juju and created a redirect
- Deleted /cloud and redirected to /openstack
- Updated links in all docs to reflect change of /cloud to /openstack (copydocs to be updated later)
- Removed /info/release-end-of-life, /financial-servies, /telecommunications and /security pages from navigation.yaml so they won't appear with secondary nav
- Added Openstack and Public cloud sections to the footer sitemap
- **Note:** contact-us and thank-you pages will need tweaks for install pages to work, but that will be another issue


## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- Navigate to http://0.0.0.0:8001/openstack and make sure it matches the [copydoc](https://docs.google.com/document/d/1eqHK0hc9v7Mn-XV36QRyNkJenTXgBQR_-RULKm5rZ5o/edit#)
- Check that all the Openstack pages are there (/index, /install, /install/try-openstack, /install/build-openstack, /managed-cloud, /partners, /storage, /foundation-cloud, /training, /contact-us, /thank-you)
- Check that redirects work for all /cloud equivalents (e.g. /cloud/managed-cloud goes to /openstack/managed-cloud)
- Check that /cloud/juju redirects to /juju
- Check that there is no secondary nav in /financial-services, /telecommunications, /security and /info/release-end-of-life
- Check that CI passes on this PR


## Issue / Card

Fixes #2664
